src/pages/settings/settings.ts

```diff
@@ -153,6 +155,13 @@ export class SettingsPage {
     if (lockMethod == 'fingerprint') this.checkFingerprint();
   }
 
+  public openMerchantDirectorySite() {
+    this.iab.create(
+      `https://bitpay.com/directory/?hideGiftCards=true`,
+      `_system`
```

Comment:
Since we're using this param to open the link (`system` browser), we don't need any additional plugin. Try to use `provider/external-link.ts`. See `home.ts` to see examples how to use it.